Sources/Kitura/ServerInfo.swift

```diff
@@ -0,0 +1,34 @@
+
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+ 
+public struct ServerInfo {
+   
+   public static let kituraVersion: String = "0.8.0" 
```

Comment:
Agreed, to that effect, I was wondering if the SPM will make build-time variables (like the #if os macro, or the #line and #file macro) but for the current git tag version. Something like #version.  I couldn't find anything like that, sadly.